l10n.js

```diff
@@ -187,7 +191,8 @@ document.webL10n = (function(window, document, undefined) {
                  .replace(/\\{/g, '{')
                  .replace(/\\}/g, '}')
                  .replace(/\\"/g, '"')
-                 .replace(/\\'/g, "'");
+                 .replace(/\\'/g, "'")
+                 .replace(/\\/g, '\n');
```

Comment:
This does not only pick up `\`s at the end of a line, but also within the line. Did you mean `.replace(/\\$/g, '\n')` ?